st3/sublime_lib/view_utils.py

```diff
@@ -90,6 +94,40 @@ def new_view(window: sublime.Window, **kwargs: Any) -> sublime.View:
     return view
 
 
+@contextmanager
+def temporarily_scratch_unsaved_views(
```

Comment:
Do we intend to expose this? If no, it should probably begin with `_` and not be included in `__all__`.

Reply:
We do need to export it here because it's used in `window_utils.close_window` (just as `clone_view` is used in the unit tests). I don't have an opinion about the underscore — given that we do have to export it, would you rather have the underscore? If so, I'll add it to both `temporarily_scratch_unsaved_views` and `clone_view`.

Reply:
An underscore means its for internal use and specifically not included in star-imports, so yes, please do. You can't not expose something in Python anyway (comparable to a "private" visibility modifier) and underscore is closest to a "protected", which is what we want here.

Reply:
Fixed.